dc/wireshark/build.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::process::Command;
+
+fn main() {
+    let flags = Command::new("pkg-config")
```

Comment:
Have you checked whether this breaks cross-compilation or direct compilation on macOS?

IIRC, the way I was making it work is just relying on these libraries being loaded by Wireshark/tshark at runtime and sharing the symbols with our code as a result (and linking with -U or some flag to ignore undefined symbols).

Reply:
I think the `-U` is what was missing. I was getting failures on macos without linking something. But let me try getting late linking working.